src/Donations/Micros/PaymentStatus/ThankyouMessage.tsx

```diff
@@ -57,7 +57,15 @@ function ThankyouMessage({
         projectName: donation.destination.name,
         location: t("country:" + donation.destination.country.toLowerCase()),
       })
-    : null;
+    : donation.project ? " " +
```

Comment:
@Shreyaschorge I think you can remove some of the code that's duplicated here. 

Perhaps store the project object as

`const donationProject = donation.destination || donation.project;`

before defining `donationProjectMessage`